packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1798,6 +1800,15 @@ class EditableText extends StatefulWidget {
   /// {@endtemplate}
   bool get selectionEnabled => enableInteractiveSelection;
 
+  /// {@template flutter.widgets.editableText.selectAllOnFocus}
+  /// Whether or not this field should highlight all text when gaining focus on
+  /// web or desktop.
+  ///
+  /// By default this will highlight the text field on web and desktop, and can
+  /// only be turned off on those two platforms.
```

Comment:
And again along with my comment that this parameter should apply equally to all platforms, you would have to remove "and can only be turned off on those two platforms."